test_requirements.txt

```diff
@@ -9,6 +9,7 @@ pytest-capturelog
 pytest-watch
 pytest-xdist
 mock
+ipdb
```

Comment:
I left it in because I thought a solution was found (that service port command-line flag). It doesn't hurt anything, and if anyone is going to use it, it's nice not to have to rebuild your Docker containers to do so.

Having said that, we can remove it if nobody thinks they're going to use it.